uint16 id
geometry_msgs/Point position
float64 distance
